### PR TITLE
Deps update and lock file check

### PR DESCRIPTION
**Pull request by dacbd**
Closes: #1080 
Closes: #1046 
Closes: #1044

### CODE REVIEW

**Review by DavidGOrtega — APPROVED** [review]

File: .github/workflows/test-deploy.yml [hunk 1]
```diff
@@ -15,6 +15,17 @@ jobs:
...
+    needs: authorize
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+      - uses: actions/setup-node@v2
+      - name: check lock
+        run: |
+          test $(cat package-lock.json | jq .lockfileVersion) == "2"
```
Comment (0x2b3bfa0) [inline .github/workflows/test-deploy.yml:28]:
```suggestion
      - run: jq --exit-status .lockfileVersion==5 < package-lock.json
```

**Review by 0x2b3bfa0 — CHANGES_REQUESTED** [review]
### _Post-mortem review_

File: .github/workflows/test-deploy.yml (hunk 1 above)
Comment (dacbd) [reply .github/workflows/test-deploy.yml:28]: rip, PR with a devcontainer?

Comment (0x2b3bfa0) [reply .github/workflows/test-deploy.yml:28]: https://github.com/iterative/cml/pull/1082

**Pull request merged**